### DISCUSSION TITLE
Configuring content exclusions for your organization

### DISCUSSION THREAD

**Discussion by GabrielaMichelon** [community/community#116670] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello,

The settings in my organization were changed according to this documentation: https://docs.github.com/en/copilot/managing-github-copilot-in-your-organization/configuring-content-exclusions-for-github-copilot
and as you can see in the picture below, we would like to disable GitHub Copilot auto completion **AND GitHub Copilot Chat** for the folder "security":
![image](https://github.com/community/community/assets/15036549/f9d9aa5d-9feb-4a66-80f5-b597b13a8ffd)

However, this is not working to disable GitHub Copilot Chat. I refresh VS Code (Reload Window command), I also see a cross over the symbol, but when you press Ctrl+I, you get GitHub Copilot Chat working. As you can see in the chat, GitHub Copilot Chat can talk about the files in this folder. Could someone implement this feature to disable GitHub Copilot Chat as well? I think this is very important and urgent as I see many people with the same problem in many forum discussions.

![image](https://github.com/community/community/assets/15036549/60ce0f36-7f41-46ad-b293-22aef4a192ec)

![image](https://github.com/community/community/assets/15036549/54216094-efe5-4ddf-9e35-a61fe0d29c95)

Thanks for your time.

**Comment by ayim** [comment]
CC: @jenverse
